unix/lib/time.mli

```diff
@@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
 
 (** Timeout operations. *)
 
-val sleep : float -> unit Lwt.t
```

Comment:
(Thought part of the point of this was to remove need for `float`s?)

On 29 June 2016 at 14:04, Thomas Leonard <notifications@github.com> wrote:

> In unix/lib/time.mli
> <https://github.com/mirage/mirage-platform/pull/168#discussion_r68941502>:
>
> > @@ -18,6 +18,6 @@ type +'a io = 'a Lwt.t
> >
> >  (** Timeout operations. *)
> >
> > -val sleep : float -> unit Lwt.t
>
> Why is this being removed? It's OK to add a new function if we need to,
> but removing the old one serves no purpose.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mirage/mirage-platform/pull/168/files/d03e97408093e46d4f779db72c2be0f1b878f770#r68941502>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAKP4KhO7u6JvbYTJqSelXadRiPhqlvYks5qQm14gaJpZM4JBE6e>
> .
>



-- 
Richard Mortier
mort@cantab.net